[2024-02-29T17:42:49.647+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_snowflake_etl.transform_data manual__2024-02-29T17:42:41.550003+00:00 [queued]>
[2024-02-29T17:42:49.662+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_snowflake_etl.transform_data manual__2024-02-29T17:42:41.550003+00:00 [queued]>
[2024-02-29T17:42:49.663+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-29T17:42:49.682+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_data> on 2024-02-29 17:42:41.550003+00:00
[2024-02-29T17:42:49.689+0000] {standard_task_runner.py:60} INFO - Started process 67 to run task
[2024-02-29T17:42:49.695+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gcs_snowflake_etl', 'transform_data', 'manual__2024-02-29T17:42:41.550003+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/GCS_to_Snowflake_DAG.py', '--cfg-path', '/tmp/tmpiw4_f2ea']
[2024-02-29T17:42:49.699+0000] {standard_task_runner.py:88} INFO - Job 303: Subtask transform_data
[2024-02-29T17:42:49.774+0000] {task_command.py:423} INFO - Running <TaskInstance: gcs_snowflake_etl.transform_data manual__2024-02-29T17:42:41.550003+00:00 [running]> on host 1753c686362f
[2024-02-29T17:42:49.888+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='gcs_snowflake_etl' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T17:42:41.550003+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-29T17:42:41.550003+00:00'
[2024-02-29T17:42:56.204+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-29T17:42:56.222+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gcs_snowflake_etl, task_id=transform_data, execution_date=20240229T174241, start_date=20240229T174249, end_date=20240229T174256
[2024-02-29T17:42:56.284+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-29T17:42:56.326+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
